packages/client/tests/functional/_utils/providers.ts

```diff
@@ -37,9 +37,9 @@ export const relationModesForFlavor = {
   [ProviderFlavors.VITESS_8]: RelationModes.PRISMA,
 } as Record<ProviderFlavors, RelationModes | undefined>
 
-export type AllProviders = { provider: Providers }[]
```

Comment:
This type wasn't imported anywhere and wasn't strictly necessary for the annotation below as the same type was inferred.